docs/development.md

```diff
@@ -9,6 +9,27 @@ Developer focused documentation should be written in this `docs/dev/` directory.
 | Playground | A collection of small `hello-world` like applications written in the technologies being used with eclipse-pass | [/eclipse-pass/playground](https://github.com/eclipse-pass/playground) | [/docs/playground.md](/docs/dev/playground.md) |
 | Pass UI | The PASS web application | [/eclipse-pass/pass-ui](https://github.com/eclipse-pass/pass-ui) | [/docs/pass-ui.md](/docs/dev/pass-ui.md) |
 
+## Development Guidelines
+
+### Committers and Contributors
+
+Only Committers on the PASS Eclipse project are able to make changes to the code repositories, however contributions from non-committers are welcome and encouraged. These contributions should be made in the form of a Pull Request (PR) on the appropriate GitHub repository.
+
+### Contribution Guidelines
```

Comment:
Do we separately want a CONTRIBUTE.md (not for this PR, but a future one) that outlines how _others_ can contribute?  I don't see this as a priority (as we don't have a _community_ yet) but something we can start to formalize (I think by _borrowing_ from other OS projects and cherry-picking what aligns with our needs).

Reply:
https://github.com/eclipse-pass/main/issues/229